ibft/IBFT.md

```diff
@@ -41,22 +41,23 @@ _Istanbul BFT is inspired by Castro-Liskov 99 [paper](http://pmg.csail.mit.edu/p
 ### Terminology
 - **`VALIDATOR`**: block validation participant - a.k.a. node.
 - **`PROPOSER`**: a block validation participant - node - chosen to propose a block in a consensus round.
-- **`ROUND`**: a consensus turn. Starts with a proposer creating a block proposal and ends with a block commitment or a round change.
+- **`ROUND`**: a consensus turn. Starts with a proposer creating a block proposal and ends with a block attestation - or a round change.
```

Comment:
Let's add a term for what is Block in IBFT context - after PROPOSER, add:

"BLOCK: data that represents the validator's duty (block proposal, attestation, aggregation) on the beacon chain network."

After this, you can change "attestation" back to "commitment" because it'll become more correct.